kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   rootPasswordSecretKeyRef:
     name: mariadb
-    key: root-password
```

Comment:
Changing this key will break all the service deploy? Example: glance, neutron, etc will do something like:

    --set endpoints.oslo_db.auth.admin.password="$(kubectl --namespace openstack get secret mariadb -o jsonpath='{.data.root-password}' | base64 -d)" \

which will look for mariadb secret with root-password. Will this work ?

Reply:
as per my understanding, if we update the password in the vault then its required to update helm charts to populate the correct password in the openstack services. I haven't validated it though.

Reply:
For infra services I don't think we can change the password references; most of them are not goverened by helm.